doc/classes/SceneTree.xml

```diff
@@ -51,7 +55,11 @@
 			<description>
 				Changes the running scene to a new instance of the given [PackedScene] (which must be valid).
 				Returns [constant OK] on success, [constant ERR_CANT_CREATE] if the scene cannot be instantiated, or [constant ERR_INVALID_PARAMETER] if the scene is invalid.
-				[b]Note:[/b] The new scene node is added to the tree at the end of the frame. You won't be able to access it immediately after the [method change_scene_to_packed] call.
+				[b]Note:[/b]
```

Comment:
We can probably just refer to the other method instead of duplicating everything. They are right next to each other in the docs anyway.

Reply:
Done.